ui/brave_ads/message_popup_view.cc

```diff
@@ -133,6 +141,13 @@ MessagePopupView::MessagePopupView(const Notification& notification)
 
 MessagePopupView::~MessagePopupView() {}
 
+#if !defined(OS_WIN) && !defined(OS_MAC)
+gfx::Rect MessagePopupView::GetVisibleFrameForPrimaryDisplay() const {
+  gfx::Size size = display::Screen::GetScreen()->GetPrimaryDisplay().size();
```

Comment:
nit: Just returning `gfx::Rect(display::Screen::GetScreen()->GetPrimaryDisplay().size())` is same.